CONTRIBUTING.md

```diff
@@ -0,0 +1,127 @@
+4. Please try using [conventional commits](https://www.conventionalcommits.org/en/v1.0.0/) for formatting your commit messages
+
+## Create a development environment (For code/dataset contributions)
+
+For code and data contributions, we recommend you creata a [conda environment](https://conda.io/projects/conda/en/latest/user-guide/tasks/manage-environments.html). If you do not have conda already installed on your system, we recommend installing [miniconda](https://docs.conda.io/en/latest/miniconda.html):
+
+```bash
+conda env create -f conda.yml  # Creates a conda env
+conda activate chemnlp  # Activate your conda environment
+```
+
+# Implementing a dataset 
+
+## Contributing a dataset 
+One of the most important ways to contribute to the ChemNLP efforts is to implement a dataset. 
+With "implementing" we mean the following: 
+
+- Take a dataset from our [awesome list](https://github.com/kjappelbaum/awesome-chemistry-datasets) (if it is not there, please add it there first, so we keep track)
+- Make an issue in this repository that you want to add this dataset (we will label this issue and assign it to you)
+- Make a PR that adds in a new folder in `data`
+  - `meta.yaml` describing the dataset 
+  - `transform.py` Python code that transforms the original dataset (linked in `meta.yaml`) into a form that can be consumed by the loader
+  - If you need additional dependencies, 
+
+
+
+The `meta.yaml` has the following structure:
+
+```yaml
+name: aquasoldb # unique identifier, we will also use this for directory names
+description: | # short description what this dataset is about
+  Curation of nine open source datasets on aqueous solubility.
+  The authors also assigned reliability groups.
+targets:
+  - id: Solubility # name of the column in a tabular dataset
+    description: Experimental aqueous solubility value (LogS) # description of what this column means
+    units: log(mol/L) # units of the values in this column (leave empty if unitless)
+    type: continuos # can be "categorical", "ordinal", "continuos"
+    names: # names for the property (to sample from for building the prompts)
+      - solubility 
+      - water solubility
+  - id: SD
+    description: Standard deviation of the experimental aqueous solubility value for multiple occurences
+    units: log(mol/L)
+    type: continuos
+    names:
+      - solubility
+      - water solubility
+      - solubility in water
+identifiers:
+  - id: InChI # column name 
+    type: InChI # can be "SMILES", "SELFIES", "IUPAC", "OTHER"
+    description: International Chemical Identifier # description (optional, except for "OTHER")
+license: CC0 1.0 # license under which the original dataset was published
+links: # list of relevant links (original dataset, other uses, etc.)
+num_points: 10000 # number of datapoints in this dataset
+  - name: dataset
+    url: https://dataverse.harvard.edu/dataset.xhtml?persistentId=doi:10.7910/DVN/OVHAW8
+    description: Original dataset
```

Comment:
above there is another description key

Reply:
The intention was that this description describes what the link is about. Will add a comment